README.md

```diff
@@ -1,15 +1,57 @@
 # cron-helm-chart
-Contains the generic helm chart that run kubernetes cron job. It is used by Prometheus & Disco project
+
+Contains the generic helm chart that run kubernetes cron job. It is used by Prometheus & Disco projects
 
 ## Usage
 
-Reference the release of the chart you want to deploy in terraform
+To deploy a k8s cronjob simply create a helm value file and refer it in the terraform code.
```

Comment:
@rbapst-tamedia It is true, but one can use the helm chart directly without Terraform. It's also true that it's not mandatory to create a `values.yaml` but you can pass Values directly via other means (`set {}` blocks, `values = [args=["foo", "bar"]]`, etc..)

what if, in the Readme we put the simplest example possible and then we create an `examples` folder with more complex like the one you showed us?